app/models/patient.rb

```diff
@@ -443,11 +482,18 @@ def latest_transfer
     where(purged: false, monitoring: true, isolation: false, continuous_exposure: false)
       .where_assoc_not_exists(:assessments, symptomatic: true)
       .where_assoc_exists(:assessments) do
-        # CAST is necessary to guarantee correct comparison between datetime and date.
-        where('CAST(assessments.created_at AS DATE) BETWEEN DATE_ADD(last_date_of_exposure, INTERVAL 7 DAY) '\
-              'AND DATE_ADD(last_date_of_exposure, INTERVAL 9 DAY)')
+        # CAST is necessary to guarantee correct comparison between datetime anfd date.
+        where(
+          'CAST(CONVERT_TZ(assessments.created_at, "UTC", patients.time_zone) AS DATE)'\
+          ' BETWEEN DATE_ADD(last_date_of_exposure, INTERVAL 7 DAY)'\
+          ' AND DATE_ADD(last_date_of_exposure, INTERVAL 9 DAY)'
+        )
```

Comment:
For this and the 10 day quarantine scopes I needed to take into account the patient's local time to calculate date in order to make the test non-sensitive to the time of day that it is run. This also benefits in that we are consistently doing a 7-9 or 10-13 days from the date of exposure (which is given in the patients' local time)